pkg/server/commands/list_objects.go

```diff
@@ -148,6 +153,7 @@ func (q *ListObjectsQuery) evaluate(
 	req listObjectsRequest,
 	resultsChan chan<- ListObjectsResult,
 	maxResults uint32,
+	resolutionMetadata *connectedobjects.ResolutionMetadata,
```

Comment:
Any reason why we add this as an argument here an not create it within the scope of this function? 

I don't see any usages that dictate this be passed in instead of created within.

Suggest changing the signature to `evaluate(...) (*ResolutionMetadata, error)` instead.

Reply:
`evaluate` fires the query but doesn't wait for the response. if we did, StreamedListObjects wouldn't work

Reply:
Yes, as @miparnisari pointed out, StreamdListObjects will not work as we lose the ability for partial results.  See [discussion](https://github.com/openfga/openfga/pull/959#discussion_r1309437519) above